index.src.html

```diff
@@ -499,8 +499,7 @@ <h3 id="referrer-policy-delivery-meta">Delivery via <{meta}></h3>
 
         <li>
           Execute the [[#set-referrer-policy]] algorithm on
-          <var>environment</var> using <var>policy</var>, if <var>policy</var>
-          is not <code>null</code>.
+          <var>environment</var> using <var>policy</var>.
```

Comment:
Nit: Recently, @tabatkins added `|var|` syntax, which you might want to use instead of `<var></var>` (as the latter is annoying to type.